changelog/check_interface_implementations.dd

```diff
@@ -0,0 +1,25 @@
+Properly check interface implementations
+
+In previous release the check to verify that a class implements all methods of
```

Comment:
In _the_ previous release, or in previous release_s_

Reply:
Release**s**.

Reply:
Fixed.